ERRORS.md

```diff
@@ -0,0 +1,116 @@
+# Common errors
+
+## Database
+
+```
+SequelizeDatabaseError: column "platformFee" does not exist
+  name: 'SequelizeDatabaseError'
+```
+
+You need to run the migration on your local db with `npm run db:migrate:dev`
+
+## Headers
+
+```
+Can't set headers after they are sent
```

Comment:
you can illustrate by linking this PR: https://github.com/OpenCollective/api/pull/244